PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -1139,8 +1139,10 @@ local function CreateCamera()
 					this.LastCameraTransform = nil
 				end
 			end
-			wait()
+			
 			setLookBehindCharacter()
+			wait()
```

Comment:
I think we can pull this in but to be safe we should be a user fast flag on this one